.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM perl:5.36.1-slim-threaded-buster
+RUN apt-get update \
+    && apt-get install \
+    build-essential \
+    ca-certificates \
+    cmake \
+    git \
+    wget \
+    locales \
+    locales-all \
+    -y -qq --no-install-recommends \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV LC_ALL=en_US.UTF-8
```

Comment:
I had problems using `cpanm` since it reported errors due to unset locales and had to include these settings. However, your other Dockerfile does not set these either and seems to work. I am not expert enough to judge whether this here is unnecessary or whether similar settings should be set in the other Dockerfile.